lib/basedriver/commands/session.js

```diff
@@ -12,12 +12,10 @@ commands.createSession = async function (desiredCapabilities, requiredCaps, capa
                                             'while one is in progress');
   }
 
-  let caps;
-  if (desiredCapabilities) {
-    caps = desiredCapabilities;
-  } else {
-    caps = processCapabilities(capabilities, this.desiredCapConstraints, this.shouldValidateCaps);
-  }
+  // If we were provided with W3C capabilities, parse those. Otherwise, fall back to MJSONWP.
+  let caps = capabilities ?
```

Comment:
i know the variables for this method were set before this PR, but coming to this code for the first time, it would have been nicer to see them named something like: `mjsonwpDesiredCapabilities`, `mjsonwpRequiredCaps`, `w3cCapabilities`.

Reply:
Yeah that's a good point. My next incoming PR for base-driver renames a lot of those.